### DISCUSSION TITLE
Can someone confirm for me this ignorePattern please

### DISCUSSION THREAD

**Discussion by CodeMedic42** [eslint/eslint#17052] [General]
I am having an issue with my ignore pattern. I am enabling a folder call ".foo". What is weird is when running in VSCode ALL the right files are linted and are showing the correct errors(I created errors to test with). However when I run eslint on the command line "eslint ." only a subset of those files are checked. I have no idea why eslint running though the command line would ignore files I explicitly enabled.

Here is a snipit of my eslintrc.json file.
{
    "ignorePatterns": [
        "!.foo/",
        ".foo/bar",
        "baz"
    ]
}

**Comment by mdjermanovic** [comment]
Hi @CodeMedic42!

> However when I run eslint on the command line "eslint ." only a subset of those files are checked.

Can you provide more details on this, which files were not checked? I just tried your configuration with a folder `.foo` locally and all the files I had there were checked.
